### DISCUSSION TITLE
End of life for 1.7.x?

### DISCUSSION THREAD

**Discussion by jak78** [PrestaShop/PrestaShop#33082] [Q&A] [answered]
Hello,
The latest Prestashop version is 8.1. 
Could you please tell me what would be the end-of-life date for older, 1.7 versions? For example: until when will the security patches be done? Same for bug fixes.
Thanks,

**Comment by Progi1984** [comment]
As explained by @matks in https://build.prestashop-project.org/news/2023/178-in-extended-support-phase/,

> This maintenance period will end when PrestaShop 9.0.0 is released. When this day comes, PrestaShop 1.7 will not be maintained anymore

**Reply by MatShir** [reply, marked as answer]
> Following PrestaShop 8.0.0 release, the 1.7.8.x branch has entered the security-only maintenance phase.

The security-only maintenance phase of the 1.7.8 will continue until the PrestaShop 9.0.0 is released. The next release 9.0.0 is not scheduled until next year.
